Sources/Overture/KeyPath.swift

```diff
@@ -117,10 +118,11 @@ public func mprop<Root, Value>(
 ///   - update: An update function for a given value.
 /// - Returns: A reference-mutable setter function.
 public func mver<Root, Value>(
-  _ keyPath: ReferenceWritableKeyPath<Root, Value>,
+  reference keyPath: ReferenceWritableKeyPath<Root, Value>,
```

Comment:
This one overload of `mver` from a reference to a reference causes ambiguity with the `inout` version when using inline closures...